apps/communications/contacts/style/app.css

```diff
@@ -637,4 +603,49 @@ html[dir="rtl"] ::-moz-placeholder {
 
 #search-list li.contact-item {
   -moz-user-select: none;
+  text-align: start;
+}
+
+/* RTL View */
+
+html[dir="rtl"] [data-type="list"].active li:after {
+  left: 0;
+  right: unset;
+}
+
+html[dir="rtl"] li button.icon-fav:before {
+  left: 0.5rem;
+  right: unset;
+}
+
+html[dir="rtl"] li button.icon-add:before {
+  left: unset;
+  right: 0.2rem
+}
+
+html[dir="rtl"] .fillflow-twocols button {
+  float: right;
+}
+
+html[dir="rtl"] .fillflow-twocols button:last-child {
+  float: left;
+}
+
+html[dir="rtl"] .fillflow-media p {
+  left: unset;
+  right: -1.5rem;
+}
+
+html[dir="rtl"] .contact-checkbox {
+  left: unset;
+  right: 0 !important;
```

Comment:
Unfortunately we can't :( Not adding !important will still make the original property a higher priority, thus it's as if we didn't fix it with this rule.